app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/OwnedListFragment.java

```diff
@@ -138,6 +138,13 @@ private void openAddEditActivity(@Nullable Book book) {
             startActivityForResult(intent, BOOK_ACTION.EDIT.getCode());
         }
     }
+    private void openDetailActivity(Book book){
+        Intent intent = new Intent(this.getActivity(), detailLibraryActivity.class);
+        intent.putExtra(BOOK_ACTION_KEY, BOOK_ACTION.VIEW);
+        intent.putExtra(VIEW_REQUEST_KEY,book);
```

Comment:
This is used for view request activity, should use a different key here

Reply:
same for RequestedListFragment. pick a name that shows that this is a key containing the book data. BOOK_DATA_KEY, BOOK_OBJECT_KEY, etc.